wordpress-mobile/WordPress-Android
# Issue/10631 material post list

Fixes #10631 

This PR moves Posts list screen to Material Theme and adds Dark Theme support. There are some code changes for cosmetic reasons.

At the time, not all screens have Dark Theme support, so the layout will look broken at most of the places.

![comparison_posts](https://user-images.githubusercontent.com/728822/70293946-be180580-1796-11ea-95dc-d50423cfb1bc.png)

To test:
- Navigate to Posts screen and make sure it looks as expected in both light and dark themes. The theme can be changed from App Settings.
 

PR submission checklist:
